inst/@sym/zeros.m

```diff
@@ -0,0 +1,85 @@
+%% Copyright (C) 2016 Lagu
+%% Copyright (C) 2016 Colin B. Macdonald
+%%
+%% This file is part of OctSymPy.
+%%
+%% OctSymPy is free software; you can redistribute it and/or modify
+%% it under the terms of the GNU General Public License as published
+%% by the Free Software Foundation; either version 3 of the License,
+%% or (at your option) any later version.
+%%
+%% This software is distributed in the hope that it will be useful,
+%% but WITHOUT ANY WARRANTY; without even the implied warranty
+%% of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See
+%% the GNU General Public License for more details.
+%%
+%% You should have received a copy of the GNU General Public
+%% License along with this software; see the file COPYING.
+%% If not, see <http://www.gnu.org/licenses/>.
+
+%% -*- texinfo -*-
+%% @documentencoding UTF-8
+%% @defmethod @@sym zeros (@var{n})
+%% @defmethodx @@sym zeros (@var{n}, @var{m})
+%% @defmethodx @@sym zeros (@var{n}, @var{m}, @var{class})
+%% Return a matrix or @var{n}-dimensional array whose elements are all 0.
```

Comment:
mm, i really don't like the idea don't doc all functions of the code, if we assume that as "not supported" we can simplify even more the code, for now i'll put that changes in other commit.